### DISCUSSION TITLE
Logging tools/strategy

### DISCUSSION THREAD

**Discussion by AntonioFalcaoJr** [AntonioFalcaoJr/EventualShop#536] [General]
What tools should we consider to implement distributed logging?

**Comment by MarcusSilvaDev** [comment]
this example: [Running Dot Net Core and Filebeat in the same Container | by Neeldeep Roy | Medium](https://medium.com/@neeldeep/running-dot-net-core-and-filebeat-in-the-same-container-a2ba0930166e)

**Comment by frankyjquintero** [comment]
Consider https://github.com/serilog-contrib/serilog-sinks-elasticsearch

https://logz.io/blog/filebeat-vs-logstash/

**Comment by roman-temekunidi** [comment]
Also, consider using [OpenTelemetry](https://opentelemetry.io/docs/). After instrumenting the app, it can be integrated with various observability tools—both proprietary and open-source ones.
